### ISSUE TITLE
Push notifications banner + in settings (out from feature preview)

### ISSUE THREAD

**Issue by bakeiro** [Couchers-org/couchers#5006]
**Current situation:**
Right now we have the enabling push notifications in the "Feature Preview" section of the site as it was new and experimental.

We're ready to move it out of there.

**Acceptance Criteria**
- Add a Banner on the Dashboard page, when they dismiss add a cookie that says pushNotificationDismissed
- Check for this cookie before showing the banner
- Maybe put it in the global message banner? Or put it in the same place.
- Add a button there with below text:

- Title: "Don't miss requests, responses or messages; can we send you push notifications?"
- Button: [Yes, please!]
                 [X] (Dismiss banner)

    * If there is no cookie, set a cookie with the `new Date()` as the value.

- Move the existing push notification settings that's in feature preview to "Account Settings" section (this allows user to also change it when it's not one month). It can be at the top above "Notification Settings".

**Comment by bakeiro** [comment]
@jesseallhands check the wording please, I'm sure title `don't miss anything` can be better 😅

**Comment by bakeiro** [comment]
note to myself: we need to test firsts notification on IOS, since seems not working properly, fix that before doing this

**Comment by bakeiro** [comment]
Ready now :)

**Comment by jesseallhands** [comment]
@bakeiro We need to be more specific about the push notifications otherwise people will just deny them outright:


"Want to receive a push notification when you get a new message or request?"
[Yes, please!]
[No]

**Comment by bakeiro** [comment]
> @bakeiro We need to be more specific about the push notifications otherwise people will just deny them outright:
> 
> "Want to receive a push notification when you get a new message or request?" [Yes, please!] [No]

yes.. since we send notification for events, friend requests and more, maybe makes sense to include it on the text? or maybe just these 2 examples is enought? (maybe the user just expects only messages and request and we have way more)
![image](https://github.com/user-attachments/assets/dcd5d78d-407e-46bb-a42c-d9ebd573078b)

**Comment by nabramow** [comment]
@aapeliv I know we talked about this, but I think it was waiting on something? Should it go back to ideas, or blocked, or can a volunteer pick it up?

**Comment by bakeiro** [comment]
@nabramow it's ready now, the `manifest.json` for iOS was merged :)

**Comment by nabramow** [comment]
Hey @aapeliv there is a line in the to-do here I think is backend. I assume we don't have this?:

`Create a new column in the user table, something like was_modal_shown, which should be false by default`

If not, I would argue rather than make it a boolean, we make it a date: `push_notification_modal_last_shown_at`.

That gives us control to show the modal for push notifications every month or so.

Does that seem doable? Or is it done already? I'll make a backend ticket if not, seems a good one for a new backend person.

**Comment by nabramow** [comment]
@jesseallhands I'm trying to refine this ticket for moving the push notifications out of feature preview. For the frontend, what do you think about?

* We show the modal on Dashboard page load once per month (could be different time range, plz suggest)
* We move the existing push notification enable/disable button out of Feature Preview and into Account Settings.

**Comment by nexagoai** [comment]
[David here]

we could even use Cookies if we don't want to create any backend ticket... [only user will see the modal every time logs in in a different device, since it won't have the cookie, but that could even make more sense]

**Comment by nabramow** [comment]
> [David here]
> 
> we could even use Cookies if we don't want to create any backend ticket... [only user will see the modal every time logs in in a different device, since it won't have the cookie, but that could even make more sense]

That's true, this actually make a lot of sense to do as a cookie. I will update the ticket!
